dev/com.ibm.ws.microprofile.health/src/com/ibm/ws/microprofile/health/internal/HealthCheckHttpResponseBuilder.java

```diff
@@ -22,26 +22,28 @@
 package com.ibm.ws.microprofile.health.internal;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 
+import javax.json.Json;
```

Comment:
Might need to follow up with Prashanth on why this file has Apache license instead of EPL license in the copyright.